### PR TITLE
Add vips

### PR DESCRIPTION
**Pull request by kojix2**
## Link

https://github.com/naqvis/crystal-vips

## Checklist
* [x] - Shard is at least 30 days old.
* [x] - Shard has CI implemented.
* [x] - Shard has daily/weekly periodic builds (ideally with Crystal latest and nightly).

### CODE REVIEW

**Review by veelenga — APPROVED** [review]
LGTM

**Comment by kojix2** [comment]
Should the name be CrystalVips rather than vips?

**Comment by veelenga** [comment]
@kojix2 ah yes, let's make it consistent please 👍

File: README.md [hunk 1]
```diff
@@ -321,6 +321,7 @@ described in [RFC 8032]
  * [ffmpeg](https://github.com/spider-gazelle/ffmpeg) - FFmpeg bindings that works with StumpyPNG to extract frames
  * [Pluto](https://github.com/phenopolis/pluto) - A fast and convenient image processing library
  * [stumpy_png](https://github.com/stumpycr/stumpy_png) - Read and write PNG images
+ * [CrystalVips](https://github.com/naqvis/crystal-vips) - Crystal bindings to [libvips](https://github.com/libvips/libvips) image processing library
```
Comment (veelenga) [inline README.md:324]:
```suggestion
 * [crystal-vips](https://github.com/naqvis/crystal-vips) - Crystal bindings to [libvips](https://github.com/libvips/libvips) image processing library
```

Comment (veelenga) [reply README.md:324]: Also, please make sure it follows alphabetical order

Comment (Blacksmoke16) [inline README.md:324]: If this lib is purely bindings to `libvips`, it should probably go under the `C Bindings` section. But if it's bindings + a Crystal API on top of it, probably should update the desc to point that out. E.g. something like: 

> Image processing library based on [libvips](https://github.com/libvips/libvips)

To denote its more than just bindings.

**Comment by veelenga** [comment]
@kojix2 please feel free to re-open if you still want to merge this.

**Pull request closed without merging**